lib/Parse/ParseDecl.cpp

```diff
@@ -4917,7 +4939,8 @@ bool Parser::isStartOfSwiftDecl(bool allowPoundIfAttributes) {
     // it's an actor declaration, otherwise, it isn't.
     do {
       consumeToken();
-    } while (isStartOfSwiftDecl(/*allowPoundIfAttributes=*/false));
+    } while (isStartOfSwiftDecl(/*allowPoundIfAttributes=*/false,
+                                /*hadAttrsOrModifiers=*/true));
```

Comment:
Can `actor` be a modifier here?

Reply:
`actor` is still declared as a (deprecated) modifier, and it's diagnosed as a error in `parseDeclModifierList()`. The diagnostic message is not great, though.

Reply:
Ah, OK. Makes sense then. Just wanted to double-check